internal/testcli/golden.go

```diff
@@ -0,0 +1,230 @@
+package testcli
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"slices"
+	"strings"
+	"testing"
+
+	"github.com/databricks/cli/internal/testutil"
+	"github.com/databricks/cli/libs/iamutil"
+	"github.com/databricks/cli/libs/testdiff"
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/iam"
+	"github.com/stretchr/testify/require"
+)
+
+var OverwriteMode = os.Getenv("TESTS_OUTPUT") == "OVERWRITE"
```

Comment:
Better to use `env.Get` in case we ever want to override this safely

Reply:
I'm not sure what would be the use case for env.Get here. This is like a test flag. It is supposed to be reading real env always.